content/concepts/kata/translations.md

```diff
@@ -5,19 +5,19 @@ sidebar: explanation-kata
 
 # Translations
 
-Codewars platform supports a variety of [programming languages](/languages/), and every kata can have more than one language version available. Users can choose any language available to solve the kata, they can also solve one kata in as many languages they wish. After gaining some experience and sufficient privileges, users are allowed to create translations to make kata available for broader group of users.
+Codewars supports a variety of [programming languages](/languages/) and every kata can have more than one language version available. Users may choose any language available to solve the kata and do so in as many languages as they wish. After gaining the required privileges, users who solved the kata are allowed to create or review translations to make the kata available for a larger audience.
 
 ## Training on a language version
```

Comment:
I just reread and "language version" sounds confusing because it's usually language's version. Not sure what's better though, so you can still merge it whey you're happy with it.

Reply:
I was thinking "translation" until it gets approved, and when it's merged into a kata, it becomes its "language version". So translation is in forks, and language version is in the kata. But I agree this term is ambiguous and easy to confuse with "language [runtime] version".

I am not sure if we really need such distinction though, so I used "translation" wherever possible.